### ISSUE TITLE
Using Fable.ReactHelper.fs with –copyExt

### ISSUE THREAD

**Issue by davidtme** [fsprojects/Fable#114]
I’m using the new copyExt flag to include the file ReactHelper.fs however when it get compiled an extra invalid *require* is being, I think it’s trying to require itself?:

```javascript
var _fableCore = require("fable-core");

var _ = require("./"); // <-- This is the extra invalid require

var _react = require("react");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Props = exports.Props = function ($exports) {
    var Prop = $exports.Prop = function Prop() {
        (0, _classCallCheck3.default)(this, Prop);
        this.Case = arguments[0];
        this.Fields = [];

        for (var i = 1; i < arguments.length; i++) {
            this.Fields[i - 1] = arguments[i];
        }
    };

    // **  Seems _ is used to access the Props/itself **

    _fableCore.Util.setInterfaces(_.Props.Prop.prototype, ["ReactHelper.Props.IHTMLProp"]); 
    var DOMAttr = $exports.DOMAttr = function DOMAttr() {
        (0, _classCallCheck3.default)(this, DOMAttr);
        this.Case = arguments[0];
        this.Fields = [];

        for (var i = 1; i < arguments.length; i++) {
            this.Fields[i - 1] = arguments[i];
        }
    };
```

**Comment by alfonsogarciacaro** [comment]
I should have checked that better, fortunately it was an easy fix. Thanks for pointing it out!

**Comment by alfonsogarciacaro** [comment]
A new version with the fix (0.2.7) has been published to npm :+1:

**Comment by davidtme** [comment]
Thanks for you help
